src/components/SetGuestUsername.vue

```diff
@@ -21,48 +21,79 @@
 
 <template>
 	<!-- Guest username setting form -->
-	<form
-		class="username-form"
-		@submit.prevent="handleChooseUserName">
-		<h3>
-			{{ t('spreed', 'Display name: ') }} <strong>{{ actorDisplayName ? actorDisplayName : t('spreed', 'Guest') }}</strong>
-			<button
-				class="icon-rename"
-				@click.prevent="handleEditUsername">
-				{{ t('spreed', 'Edit') }}
-			</button>
-		</h3>
-		<div
-			v-if="isEditingUsername"
-			class="username-form__wrapper">
-			<input
-				ref="usernameInput"
-				v-model="guestUserName"
-				:placeholder="t('spreed', 'Guest')"
-				class="username-form__input"
-				type="text"
-				@keydown.enter="handleChooseUserName"
-				@keydown.esc="isEditingUsername = !isEditingUsername">
-			<button
-				class="username-form__button"
-				type="submit">
-				<div class="icon-confirm" />
-			</button>
-		</div>
-	</form>
+	<div>
+		<form
+			class="username-form"
+			@submit.prevent="handleChooseUserName">
+			<h3>
+				{{ t('spreed', 'Display name: ') }} <strong>{{ actorDisplayName ? actorDisplayName : t('spreed', 'Guest') }}</strong>
+				<button
+					class="icon-rename"
+					@click.prevent="handleEditUsername">
+					{{ t('spreed', 'Edit') }}
+				</button>
+			</h3>
+			<div
+				v-if="isEditingUsername"
+				class="username-form__wrapper">
+				<input
+					ref="usernameInput"
+					v-model="guestUserName"
+					:placeholder="t('spreed', 'Guest')"
+					class="username-form__input"
+					type="text"
+					@keydown.enter="handleChooseUserName"
+					@keydown.esc="isEditingUsername = !isEditingUsername">
+				<button
+					class="username-form__button"
+					type="submit">
+					<div class="icon-confirm" />
+				</button>
+			</div>
+		</form>
+		<Modal v-if="modal" :canClose="false">
+			<form
+				class="username-form"
+				@submit.prevent="handleChooseUserName2">
+				<div
+					class="username-form__wrapper"
+					style="display: block">
+					<h3>
+						{{ t('spreed', 'Display name: ') }}
```

Comment:
trailing spaces don't work in translations. I guess you need to figure this out with CSS instead.

Reply:
There's a space in translations:
https://github.com/nextcloud/spreed/blob/4030de49238322826dfbe2b1dc160943545c53c4/l10n/de.js#L918

Reply:
yeah but as you can see the german version is missing it already. They can not add trailing whitespace on transifex as they don't see it would need being there

Reply:
As I see it's in all languages:
https://github.com/nextcloud/spreed/blob/4030de49238322826dfbe2b1dc160943545c53c4/l10n/br.js#L684
https://github.com/nextcloud/spreed/blob/4030de49238322826dfbe2b1dc160943545c53c4/l10n/es.js#L901
https://github.com/nextcloud/spreed/blob/4030de49238322826dfbe2b1dc160943545c53c4/l10n/ru.js#L897
And if I delete a space - it doesn't translate.

Reply:
Translations will come afterwards automatically

Reply:
Ah, you only intend it, I will fix it then

Reply:
Raised https://github.com/nextcloud/spreed/issues/5500